pkg/otlp/logs/logs_translator.go

```diff
@@ -156,7 +155,7 @@ func Transform(lr plog.LogRecord, res pcommon.Resource, logger *zap.Logger) data
 	if lr.Timestamp() != 0 {
 		// we are retaining the nano second precision in this property
 		l.AdditionalProperties[otelTimestamp] = strconv.FormatInt(lr.Timestamp().AsTime().UnixNano(), 10)
-		l.AdditionalProperties[ddTimestamp] = lr.Timestamp().AsTime().Format(time.RFC3339)
+		l.AdditionalProperties[ddTimestamp] = lr.Timestamp().AsTime().Format("2006-01-02T15:04:05.000Z")
```

Comment:
The previous value `time.RFC3339 = "2006-01-02T15:04:05Z07:00"` has the suffix of `07:00` in its format string. Is there any reason why this suffix is omitted here?

Reply:
Good question!  I have added the timezone format in to mirror `time.RFC3339`.

(And fixed the changelog, gah.)